azext_iot/iothub/_help.py

```diff
@@ -309,3 +309,32 @@ def load_iothub_help():
         type: command
         short-summary: Upload a local file as a device to a pre-configured blob storage container.
     """
+
+    helps["iot edge hierarchy"] = """
```

Comment:
We should double-check the _official terminology_ for this topic to ensure alignment...I've seen both "hierarchy" and "topology" being used.